bestie/server/main.go

```diff
@@ -25,6 +25,7 @@ import (
 var (
 	logger      = log.New(os.Stdout, "bestie: ", log.Ldate|log.Ltime|log.Lmicroseconds|log.LUTC|log.Lshortfile|log.Lmsgprefix)
 	isDebugMode bool // Set this to true to enable certain debug behaviors (e.g. special log messages).
+	maxFileSize int  = (20 * 1024 * 1024)
```

Comment:
Given the current RAM limit - maybe 5MB is a good starting place? 4x as many concurrent events before OOM

Reply:
Changed from 20 MB to 5 MB.